en/docs/learn/consume-api/manage-subscription/advanced-topics/adding-an-api-subscription-tier-update-workflow.md

```diff
@@ -44,7 +39,7 @@ First, enable the API subscription update workflow for **Approval Workflow Execu
     
     ![subscription_update_new_tier_request]({{base_path}}/assets/img/learn/subscription-update-new-tier-request.png)
     
-6.  Sign in to the Admin Portal ( `https://<Server Host>:9443/admin` ), list all the tasks for API subscription update from **Tasks** --> **Subscription Update** and click on  approve or reject to approve or reject workflow pending request.
+6.  Sign in to the Admin Portal ( `https://<Server Host>:9443/admin` ), list all the tasks for API subscription update from **Tasks** --> **Subscription Update** and click on  approve (or reject) to approve (or reject) the workflow pending request.
```

Comment:
Shall we remove the unnecessary spaces in this line. 
- After (
- Before )
- After "and click on"

Reply:
There s no spaces After ( & Before ) that shows to me. Could you please recheck and see if it still is visible?

Reply:
What I was saying is that ( `https://<Server Host>:9443/admin` ) should be (`https://<Server Host>:9443/admin`) without adding a space before and after ' '

Reply:
I see. Fixed it there.